spec/controllers/people/skills_controller_spec.rb

```diff
@@ -21,6 +22,20 @@
         expect(junit_attrs['person_id']).to eq (ken.id)
         json_object_includes_keys(junit_attrs, keys)
       end
+
+      it 'returns Rails skills' do
```

Comment:
Er gibt hier immer noch people_skills zurück und nicht skills oder? naming wäre dann irgendwie `returns people_skills for Rails Skill`